src/main/java/jenkins/plugins/openstack/compute/SlaveOptionsDescriptor.java

```diff
@@ -551,7 +714,22 @@ public void calcFillSettings(String field, Map<String, Object> attributes) {
             }
         }
 
-        attributes.put("fillDependsOn", Joiner.on(' ').join(deps));
+        if (!deps.isEmpty()) {
+            attributes.put("fillDependsOn", Joiner.on(' ').join(deps));
+        }
+    }
+
+    private static boolean haveServerDetails(String endPointUrl, String identity, String credential, String zone) {
```

Comment:
`haveAuthDetails`, please. Not to be confused with server to be provisioned its details are processed here as well.